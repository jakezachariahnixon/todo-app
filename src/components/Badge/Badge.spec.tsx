import React from "react";
import renderer from "react-test-renderer";
import { cleanup } from "@testing-library/react";

import { Badge } from "./index";

describe("Badge", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Badge>Click Me</Badge>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});

afterEach(cleanup);
