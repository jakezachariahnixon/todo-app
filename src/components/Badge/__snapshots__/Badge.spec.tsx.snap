// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Badge renders correctly 1`] = `
<div
  className="c-gprliu c-gprliu-cmPWsT-color-blue"
>
  Click Me
</div>
`;
